gdal_subsetter/transform.py

```diff
@@ -668,23 +668,25 @@ def _regrid(infile, outfile, resampling_mode='bilinear', ref_crs=None, ref_box=N
     def recolor(self, layerid, srcfile, dstdir):
```

Comment:
It'd be nice to encapsulate what this method does in a documentation string. (It might mean some of the in-line comments become unnecessary)